docker-compose.yml

```diff
@@ -9,33 +9,42 @@ services:
     links:
       - postgres
       - redis
+    depends_on:
+      - postgres
+      - redis
     env_file:
       - docker/miaou-default.env # define default environment variables
       - docker/miaou.env # allow to override or add local environment variables
     # Uncomment this to trigger automatic application restart when a file is modified (consider adding the option "-L" if you use shared folders)
 #    command: "nodemon --use_strict main.js"
     # Uncomment this to mount watched directory/files listed below (to work with "nodemon" above)
-#    volumes:
```

Comment:
Just delete the whole thing? Commented out code is rarely a good idea.

Reply:
This was already here ! I guess I just failed to recomment it properly?

Reply:
Nah, when you come across ugly code, just delete it. Legacy is a thing, but let's move forward :)